index.html

```diff
@@ -59,6 +59,9 @@ <h3 class="challenger-intext-label">guess</h3>
               <article>
                 <input type="text" class="player-input-box challenger-1-guess-input" id="one-guess" />
               </article>
+              <article>
+                <div class = "range-error-one"></div>
+              </article>
```

Comment:
I don't think the article is necessary, and I would probably make the error message a <p> instead of a <div>